tests/integration/conftest.py

```diff
@@ -184,3 +184,12 @@ def _create_user():
         return UserFactory()
 
     return _create_user
+
+
+@pytest.fixture(scope="function")
+def elastic(app):
+    with app.app_context():
+        _es = app.extensions["invenio-search"]
+        list(_es.delete(ignore=[404]))
+        list(_es.create(ignore=[400]))
+        list(_es.put_templates(ignore=[400]))
```

Comment:
why we need this fixture? we have ``es_clear`` https://pytest-invenio.readthedocs.io/en/latest/api.html#pytest_invenio.fixtures.es_clear :)

Reply:
Because `es_clear` requires `es` and `es` requires `appctx` which requires `base_app` which requires `create_app` which should be scope 'module' but for async tests our app need to be `session` scoped. It's a little mess with this es_clear...

Reply:
but this is not the ``async`` tests :/

Reply:
You are right... My mistake.